### PR TITLE
Docs migrate to Napoleon -> Nelson

### PR DESCRIPTION
**Pull request by psav**
Docs migrate to Napoleon

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 5e116a03cc46e6f6b1aa60a3b68dd272cf945382:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Comment by seandst** [comment]
`modulename/__init__.py` should just be `modulename.py`.

You'd mentioned that we need sphinx 1.3b2 for this, but I don't see that reflected in requirements.

**Comment by seandst** [comment]
Also, the existing sphinx extension lives in utils, so I think this one probably should, as well (`utils/nelson.py`)

**Comment by psav** [comment]
Updated as requested.

**Comment by dajoRH** [comment]
Lint report for commit 6a89aef005bb257c0264861ec28267e028948ec6:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: requirements.txt [hunk 1]
```diff
@@ -30,6 +29,7 @@ selenium
 slumber
 multimethods.py
 sphinx-rtd-theme
+Sphinx==1.3b2
```
Comment (seandst) [inline requirements.txt]: /me thinks ">=1.3b2" would be better, so we aren't stuck on the beta

**Comment by dajoRH** [comment]
Lint report for commit 9361167044a78a9ecb4ab8befd9085192849fb80:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

**Pull request merged**